composer.lock

```diff
@@ -7376,4 +7376,4 @@
     "prefer-lowest": false,
     "platform": [],
     "platform-dev": []
-}
```

Comment:
😭

Reply:
Yeah, my IDE has been doing that on some files. I'll check it out.